docs/api-docs/channels/channels-extending-existing.md

```diff
@@ -3,19 +3,20 @@
 <div class="otp" id="no-index">
 
 ### On this Page

```

Comment:
Remove the phrase "Step #:" from the titles, and adjust the capitalization of the headers accordingly.

Reply:
Those were already in the article and not a part of this update, so shouldn't hold up the pull request.

Reply:
This correction is in line with the description of the changes you made throughout the document (i.e. "applied style guide updates").

Reply:
Okay. I removed that part.

Reply:
Will address style guide updates in [new ticket](https://jira.bigcommerce.com/browse/DEVDOCS-2360).